roles/ovirt-external-providers/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Add external providers
+  ovirt_external_providers:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ item.name }}"
+    state: "{{ item.state }}"
+    type: "{{ item.type }}"
+    url: "{{ item.url }}"
+    username: "{{ item.username }}"
+    password: "{{ item.password }}"
+    tenant: "{{ item.tenant }}"
+    auth_url: "{{ item.auth_url }}"
```

Comment:
Yes i need to pass 
    name: "{{ item.name }}"
    state: "{{ item.state | default('present') }}"
    type: "{{ item.type }}"
    url: "{{ item.url }}"
    username: "{{ item.username  }}"
    password: "{{ item.password  }}"
    tenant: "{{ item.tenant   }}"
    auth_url: "{{ item.auth_url }}"
    data_center: "{{ item.data_center | default(omit) }}"
 
for the ovirt public glance i passed dummy data on those parameters and it added it. 

how to proceed from here?